Temporarily revert https://github.com/yuzu-emu/yuzu/commit/645961613fd5
until devel/simpleini installs CMake config and/or pkg-config files.

--- CMakeModules/FindSimpleIni.cmake.orig	2024-01-26 15:23:08 UTC
+++ CMakeModules/FindSimpleIni.cmake
@@ -2,20 +2,18 @@
 #
 # SPDX-License-Identifier: GPL-3.0-or-later
 
+find_path(SimpleIni_INCLUDE_DIR SimpleIni.h)
+
 include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(SimpleIni
+    REQUIRED_VARS SimpleIni_INCLUDE_DIR
+)
 
-find_package(SimpleIni QUIET CONFIG)
-if (SimpleIni_CONSIDERED_CONFIGS)
-    find_package_handle_standard_args(SimpleIni CONFIG_MODE)
-else()
-    find_package(PkgConfig QUIET)
-    pkg_search_module(SIMPLEINI QUIET IMPORTED_TARGET simpleini)
-    find_package_handle_standard_args(SimpleIni
-        REQUIRED_VARS SIMPLEINI_INCLUDEDIR
-        VERSION_VAR SIMPLEINI_VERSION
+if (SimpleIni_FOUND AND NOT TARGET SimpleIni::SimpleIni)
+    add_library(SimpleIni::SimpleIni INTERFACE IMPORTED)
+    set_target_properties(SimpleIni::SimpleIni PROPERTIES
+        INTERFACE_INCLUDE_DIRECTORIES "${SimpleIni_INCLUDE_DIR}"
     )
 endif()
 
-if (SimpleIni_FOUND AND NOT TARGET SimpleIni::SimpleIni)
-    add_library(SimpleIni::SimpleIni ALIAS PkgConfig::SIMPLEINI)
-endif()
+mark_as_advanced(SimpleIni_INCLUDE_DIR)
